src/website/src/app/documentation/demos/stepper/design-stepper.demo.html

```diff
@@ -0,0 +1,86 @@
+<p>
+  Each panel representing each step will contain a title and a description that describes the entire workflow at a
+  glance. After completing each step, collapsed panels then replace step descriptions with summaries describing what the
+  user has accomplished.
+</p>
+
+<p>
+  Here is when you might want to use a stepper over normal forms or wizards:
+</p>
+
+<ul>
+  <li>Complex and long work processes that may not fit in one viewport</li>
+  <li>When the user would need to see the forms in-line with the rest of the page content</li>
+  <li>If users need more detail than step titles to understand the process.</li>
+  <li>If summaries of previous steps are helpful in informing next steps.</li>
+</ul>
+<h3>Style</h3>
+<img src="assets/images/documentation/stepper/stepper-style.png" alt="" class="clrweb-DoxMedia-img" />
+
+<p>
+  Steppers are composed of two sections- the stepper header and input section.
```

Comment:
nit: whitespace between `sections` and `-`

Reply:
Fixed, I think it should be a comma anyways.